plugin/virustotal/vtgrep.py

```diff
@@ -14,12 +14,11 @@
 __author__ = 'gerardofn@virustotal.com'
 
 import binascii
-import ida_kernwin
-import idaapi
-import idautils
-import idc
 import logging
+from virustotal.vt_ida.disassembler import *
```

Comment:
import * is a fairly bad pattern -- please import each class or function explicitly